editor.planx.uk/src/@planx/components/FileUpload/Editor.tsx

```diff
@@ -62,7 +62,7 @@ function Component(props: any) {
           </InputRow>
           <InputRow>
             <Input
-              // required
+              required
```

Comment:
I agree this should be correct behavior - but I wonder if this will break any discretionary submission services like building control forms or something? Hopefully not / easy enough to fix forward if we find it does I guess? I don't necessarily think we should be putting as much proactive effort into find/replace etc outside of "standard" services?